### PR TITLE
[HB-5551] Define Adapter Configuration JSON Schema

### PR DESCRIPTION
**Pull request by SCastanedaMunoz**
# Description

This is the base AdapterConfig.json Schema based on the currently released adapters, this PR is only in charge of defining the structure we will be reading from Unity. Reading the File and Adding Dependency.xml files will be added into a different PR.

### CODE REVIEW

File: AdapterConfig.json [hunk 1]
```diff
@@ -0,0 +1,400 @@
...
+            }
+        },
+        {
+            "id" : "admob",
+            "name" : "AdMob",
+            "android" : {
+                "name" : "chartboost-mediation-adapter-admob",
+                "sdk" : "com.google.android.gms:play-services-ads",
+                "latest" : "21.5.0",
+                "versions" : [
```
Comment (cb-jpadilla) [inline AdapterConfig.json:31]: Is this JSON Schema going to be manually updated and will the latest versions always be the first element in the arrays?

File: AdapterConfig.json [hunk 2]
```diff
@@ -0,0 +1,400 @@
...
+                "sdk" : "Yahoo-Mobile-SDK",
+                "latest" : "1.4.0",   
+                "versions" : [
+                    "1.4.0",
+                    "1.3.0"
+                ]
+            }
+        }
+    ]
+}
```
Comment (cb-jpadilla) [inline AdapterConfig.json:400]: EOL

**Review by cb-jpadilla — COMMENTED** [review]
Looks good, but let's add an EOL.

**Review by pleasesavemycat — APPROVED** [review]
lgtm

**Pull request merged**